bundles/org.openhab.binding.mybmw/src/main/resources/OH-INF/config/bridge-config.xml

```diff
@@ -14,6 +14,11 @@
 			<description>MyBMW Password</description>
 			<context>password</context>
 		</parameter>
+		<parameter name="hcaptchatoken" type="text" required="true">
+			<label>Captcha-Token</label>
+			<description>Captcha-Token for login (see https://bimmer-connected.readthedocs.io/en/stable/captcha.html)</description>
+			<context>hcaptchatoken</context>
```

Comment:
Is this a known context? This is used in the UI, but I don’t know about this value.

Reply:
The Captcha token was introduced recently by BMW and now the binding is not working anymore. It was the easiest solution to add the param into the bridge config

Reply:
Yes, but I think the context field is not required and does not do anything.

Reply:
Ok I just copied it from the password configuration

Reply:
Indeed, but password is a known context. When the parameter has a password context, the field content will be hidden by default. If you want this token to be hidden, you can equally set it to context password,  ut I don’t think that’s needed here.